integration/resources/templates/combination/connector_appsync_to_eventbus.yaml

```diff
@@ -0,0 +1,173 @@
+      ApiId: !GetAtt AppSyncApi.ApiId
+      TypeName: Query
+      FieldName: sayHello
+      DataSourceName: !GetAtt AppSyncEventBusDataSource.Name
+      RequestMappingTemplate: |
+        {
+          "version" : "2018-05-29", 
+          "operation" : "PutEvents",
+          "events" : [{}]
+        }
+      ResponseMappingTemplate: |
+        #if($ctx.error)
+          $util.error($ctx.error.message, $ctx.error.type, $ctx.result)
+        #end
+          $util.toJson($ctx.result)
+
+  Connector:
+    Type: AWS::Serverless::Connector
+    Properties:
+      Source:
+        Id: AppSyncEventBusDataSource
+      Destination:
+        Id: EventBus
+      Permissions:
+      - Write
+
+  ApiKey:
+    Type: AWS::AppSync::ApiKey
+    Properties:
+      ApiId: !GetAtt AppSyncApi.ApiId
+
+  TriggerFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Role: !GetAtt EventBridgeRole.Arn
+      Environment:
+        Variables:
+          API_KEY: !GetAtt ApiKey.ApiKey
+          GRAPHQL_URL: !GetAtt AppSyncApi.GraphQLUrl
+          EventBusName: !Ref EventBus
+      Runtime: nodejs16.x
+      Handler: index.handler
+      InlineCode: |
+        const https = require("https");
+        const AWS = require('aws-sdk');
+
+        exports.handler = async () => {
+          const queries = {
+            sayHello: /* GraphQL */ `
+              query {
+                sayHello
+              }
+            `,
+          };
+
+          const fetch = async (url, options) =>
+            new Promise((resolve, reject) => {
+              const req = https.request(url, options, (res) => {
+                const body = [];
+                res.on("data", (chunk) => body.push(chunk));
+                res.on("end", () => {
+                  const resString = Buffer.concat(body).toString();
+                  resolve(resString);
+                });
+              });
+              req.on("error", (err) => {
+                reject(err);
+              });
+              req.on("timeout", () => {
+                req.destroy();
+                reject(new Error("Request time out"));
+              });
+              req.write(options.body);
+              req.end();
+            });
+
+          const makeRequest = async (queryName) => {
+            const options = {
+              method: "POST",
+              headers: {
+                "x-api-key": process.env.API_KEY,
+              },
+              body: JSON.stringify({ query: queries[queryName] }),
+              timeout: 10000, // ms
+            };
+
+            const response = await fetch(process.env.GRAPHQL_URL, options);
+            let body = JSON.parse(response);
+            const data = body.data?.[queryName];
+            
+            if (body.errors !== undefined) {
+              throw JSON.stringify(body.errors);
+            }
+            
+            return body.data;
+          };
+
+          const sayHelloResponse = await makeRequest("sayHello");
```

Comment:
Is it okay to call it before PutEvent()?

Reply:
Related to https://github.com/aws/serverless-application-model/pull/3311/files#r1300549275, will be resolved with verification of event being in the eventBus.